apps/multisig/src/components/TransactionSidesheet/TransactionSidesheetFooter.tsx

```diff
@@ -68,6 +79,39 @@ export const SignerCta: React.FC<{
     return !Object.values(t.approvals).find(v => v === true)
   }, [t])
 
+  const multisigHasEnoughBalance = useMemo(() => {
+    // No need to check for multisig balance if not going to execute the transaction
+    if (asDraft || !readyToExecute || existentialDepositLoadable.state === 'loading') return true
+
+    const [multiSigTokenBalance] =
+      balances?.find(({ address, chainId }) => {
+        const parsedAddress = Address.fromSs58(address)
+        return (
+          parsedAddress &&
+          parsedAddress.isEqual(multisig.proxyAddress) &&
+          chainId === t.multisig.chain.squidIds.chainData
+        )
+      }) || []
+
+    const availableBalance = multiSigTokenBalance?.transferable.planck ?? 0n
```

Comment:
There's 2 issues with this:
- it should also check for native token only, this may sum the transferrable tokens that are non native (e.g. usdc on asset hub)
- balances.find() does not return an array. It returns the Balances class

Here's how you could address these bugs:

```suggestion
    const availableBalance =
      balances?.find(({ address, chainId, source }) => {
        const parsedAddress = Address.fromSs58(address)
        return (
          parsedAddress &&
          parsedAddress.isEqual(multisig.proxyAddress) &&
          source === 'substrate-native' &&
          chainId === t.multisig.chain.squidIds.chainData
        )
      }).sum.planck.transferable ?? 0n
```